fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordCursorResult.java

```diff
@@ -200,6 +200,14 @@ public int hashCode() {
         }
     }
 
+    /**
+     * Returns true if the cursor has reached its end but a continuation is still available (i.e. the source is not yet exhausted).
```

Comment:
`true` -> `{@code true}`.

Perhaps it's clearer to change "a continuation is still available" to "the continuation is not an end continuation" because a continuation is always "available" (it might just be `END`)?